x/auth/client/cli/tx_sign.go

```diff
@@ -60,11 +60,11 @@ account key. It implies --signature-only.
 }
 
 func makeSignBatchCmd() func(cmd *cobra.Command, args []string) error {
```

Comment:
This function is wayyyy to large for naked returns. If using named return value(s), please avoid naked returns and explicitly return the value(s).